integration_tests/e2e/goal/review.cy.ts

```diff
@@ -94,6 +94,47 @@ context('Review goal(s)', () => {
     Page.verifyOnPage(AddNotePage)
   })
 
+  it('should be able to create another goal from the Review screen', () => {
+    // Given
+    const prisonNumber = 'G6115VJ'
+    cy.signIn()
+    cy.visit(`/plan/${prisonNumber}/view/overview`)
+    const overviewPage = Page.verifyOnPage(OverviewPage)
+
+    const createGoalPage = overviewPage.clickAddGoalButton()
+    createGoalPage //
+      .setGoalTitle('Learn French')
+      .submitPage()
+
+    const addStepPage = Page.verifyOnPage(AddStepPage)
+    addStepPage //
+      .setStepTitle('Book French course')
+      .setStepTargetDateRange('ZERO_TO_THREE_MONTHS')
+      .submitPage()
+
+    const addNotePage = Page.verifyOnPage(AddNotePage)
+    addNotePage.setNote("Pay close attention to Chris' behaviour during classes")
+    addNotePage.submitPage()
+
+    // When
+    const reviewPage = Page.verifyOnPage(ReviewPage)
+    reviewPage.addAnotherGoal()
+
+    createGoalPage //
+      .setGoalTitle('Learn Spanish')
+      .submitPage()
+
+    addStepPage //
+      .setStepTitle('Book Spanish course')
+      .setStepTargetDateRange('ZERO_TO_THREE_MONTHS')
+      .submitPage()
+
+    addNotePage.submitPage()
```

Comment:
This test is good in that it exercises the navigation of clicking the add another goal button and going back round the journey again, but I think there's a few things we can do to improve it, specifically around what we are asserting.

I think it would be good to assert that once we are on the ReviewGoal page for the first time (ie. at the end of the // Given section and before the // When section) we assert that we are currently displaying 1 goal. Maybe we could also assert the title of the goal (perhaps?), but at a minimum I think we should assert we have added and are displaying 1 goal.

Once we've then added the 2nd goal and have arrived back on the ReviewGoal page (ie. your // Then block) I think we need to assert we are now displaying 2 goals (along with their titles if you chose to do that above)

I also think that the page variables you are using in the // When block (createGoalPage, addStepPage, addNotePage etc) are the object references from the // Given block and possibly might not reference the "2nd display" of that page (if that makes sense). I think you'd be better off re-assigning them, and using a `let` rather than a `const`. Something like this:

```
    let createGoalPage = overviewPage.clickAddGoalButton()

    let addStepPage = createGoalPage //
      .setGoalTitle('Learn French')
      .submitPage()

    let addNotePage = addStepPage //
      .setStepTitle('Book French course')
      .setStepTargetDateRange('ZERO_TO_THREE_MONTHS')
      .submitPage()

    let reviewPage = addNotePage // 
      .setNote("Pay close attention to Chris' behaviour during classes")
      .submitPage()

    // assert here that there is 1 goal on the review page, optionally with its goal title

    // When
    reviewPage.addAnotherGoal()

    createGoalPage = Page.verifyOnPage(CreateGoalPage)
    addStepPage = createGoalPage //
      .setGoalTitle('Learn Spanish')
      .submitPage()

    addNotePage = addStepPage //
      .setStepTitle('Book Spanish course')
      .setStepTargetDateRange('ZERO_TO_THREE_MONTHS')
      .submitPage()

    reviewPage = addNotePage.submitPage()

    // Then
    // assert here that there are 2 goals on the review page, optionally with their goal titles  
})
```